Assets/SteamVR_Unity_Toolkit/Scripts/SteamVR_HeadsetCollisionFade.cs

```diff
@@ -30,6 +30,13 @@ protected HeadsetCollisionEventArgs SetHeadsetCollisionEvent(Collider collider,
     }
 
     protected void Start () {
+        //Make sure that there is a SteamVR_Fade script available.
+        SteamVR_Fade fadeScript = gameObject.GetComponentInChildren<SteamVR_Fade>();
+        if (fadeScript == null)
```

Comment:
May be worth just doing the if inline like so:
`if (gameObject.GetComponentInChildren<SteamVR_Fade>() == null)`

Rather than assigned to a variable then never use it anywhere other than the if statement.